entrypoint.sh

```diff
@@ -12,6 +12,8 @@ SOURCE_DIRECTORY=${GITHUB_WORKSPACE}/$INPUT_SOURCE
 DESTINATION_DIRECTORY=${GITHUB_WORKSPACE}/$INPUT_DESTINATION
 PAGES_GEM_HOME=/src/gh/pages-gem
 GITHUB_PAGES=$PAGES_GEM_HOME/bin/github-pages
+JEKYLL_GITHUB_TOKEN=$GITHUB_TOKEN
+PAGES_REPO_NWO=$GITHUB_REPOSITORY
```

Comment:
I think this should work since it would just pull the environment variables from the actions runtime?